ADALiOS/ADALiOS/ADWebRequest.m

```diff
@@ -237,9 +241,12 @@ - (NSURLRequest *)connection:(NSURLConnection *)connection willSendRequest:(NSUR
 {
 #pragma unused(connection)
 #pragma unused(redirectResponse)
+    NSURL* requestURL = [request URL];
+    NSURL* modifiedURL = [ADHelpers addClientVersionToURL:requestURL];
+    if (modifiedURL == requestURL)
```

Comment:
Does it say equal for "http://somesite.com" and "http://somesite.com/" ? You may need to normalize the url before comparing.